tests/production_pause_vote_timeout.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+import os
+import shutil
+import signal
+import time
+
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.Node import BlockType
+
+####################################################################################
+# production_pause_vote_timeout
+# Test production-pause-vote-timeout works as expected.
+#
+# Setup:
+#
+# Use five nodes in an hourglass topology. The center node is a relay node that
+# initially has vote-threads enabled. The other 4 peripheral nodes are:
+#
+# node0: Enables block production for producera and has the finalizer key with
+#        description of producera. Has vote-threads enabled. Connect to the center node.
+# node1: Enables block production for producerb and has the finalizer key with
+#        description of producerb. Has vote-threads enabled. Connect to the center node
+#        and Node1.
+# producercNode: Enables block production for producerc.
```

Comment:
Must have vote threads enabled, right? And connect to `finalizercNode` and center node?